### PR TITLE
20759069: added Partner Banner

### PR DESCRIPTION
**Pull request by amindu00**
must add delay to the loop otherwise its weird.

### CODE REVIEW

File: src/components/container/App/App.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
+import PartnerBanner from '../PartnerBanner/PartnerBanner';
 import './App.scss';
 
 function App() {
   return (
     <div className="App">
+     
+      <PartnerBanner />
```
Comment (jasper-prgh) [inline src/components/container/App/App.js]: Remove this from here please, we will readd this later

File: src/components/container/PartnerBanner/PartnerBanner.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      <div>
+        <img alt="start" />
+        {BANNERS.map((b, i) => <img key={i} src={b.src} alt={b.name} />)}
+        <img alt="end" />
+      </div>
+    </div>
+  )
+}
+
+export default PartnerBanner
```
Comment (jasper-prgh) [inline src/components/container/PartnerBanner/PartnerBanner.js]: ; missing

File: src/components/container/PartnerBanner/PartnerBanner.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import './PartnerBanner.scss'
+import Logo from './assets/partnerLogos/prgh-dark-main.svg';
+//this method gonna leads to so many imports so this is the right method?
+
+function PartnerBanner() {
+  const BANNERS = Array(10).fill({ name: "PRGH", src: Logo })  //dev version to add multiple logos
```
Comment (jasper-prgh) [inline src/components/container/PartnerBanner/PartnerBanner.js]: ;

File: src/components/container/PartnerBanner/PartnerBanner.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import './PartnerBanner.scss'
```
Comment (jasper-prgh) [inline src/components/container/PartnerBanner/PartnerBanner.js]: ;

**Review by jasper-prgh — CHANGES_REQUESTED** [review]

File: src/components/container/App/App.js [hunk 5]
```diff
@@ -2,7 +2,8 @@ import './App.scss';
 
 function App() {
   return (
-    <div className="App">
+    <div className="App">
```
Comment (jasper-prgh) [inline src/components/container/App/App.js]: revert all changes in the app.js please. lets pevent any merge conflicts

File: src/components/container/PartnerBanner/PartnerBanner.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './PartnerBanner.scss';
+import Logo from './assets/partnerLogos/prgh-dark-main.svg';
+//this method gonna leads to so many imports so this is the right method?
```
Comment (jasper-prgh) [inline src/components/container/PartnerBanner/PartnerBanner.js]: remove comment

File: src/components/container/PartnerBanner/PartnerBanner.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './PartnerBanner.scss';
+import Logo from './assets/partnerLogos/prgh-dark-main.svg';
+//this method gonna leads to so many imports so this is the right method?
+
+function PartnerBanner() {
+  const BANNERS = Array(10).fill({ name: "PRGH", src: Logo }); //dev version to add multiple logos
```
Comment (jasper-prgh) [inline src/components/container/PartnerBanner/PartnerBanner.js]: remove comment and change naming to `banners`

File: src/components/container/PartnerBanner/PartnerBanner.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+//this method gonna leads to so many imports so this is the right method?
+
+function PartnerBanner() {
+  const BANNERS = Array(10).fill({ name: "PRGH", src: Logo }); //dev version to add multiple logos
+
+  return (
+    <div className='PartnerBanner'>
+      <div>
+        <img alt="start" />
+        {BANNERS.map((b, i) => <img key={i} src={b.src} alt={b.name} />)}
```
Comment (jasper-prgh) [inline src/components/container/PartnerBanner/PartnerBanner.js]: formatting

**Review by jasper-prgh — CHANGES_REQUESTED** [review]

File: src/components/container/PartnerBanner/PartnerBanner.js (hunk 8 above)
Comment (amindu00) [reply src/components/container/PartnerBanner/PartnerBanner.js]: start
this is something I put manually to see the starting imaging and ending image to check every image is looping.
if you meant the indent in line 13

File: src/components/container/PartnerBanner/PartnerBanner.js [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+import './PartnerBanner.scss';
+import Logo from './assets/partnerLogos/prgh-dark-main.svg';
+
+function PartnerBanner() {
+  const banners = Array(10).fill({ name: "PRGH", src: Logo });
+
+  return (
+    <div className='PartnerBanner'>
+      <div>
+        <img alt="start" />
```
Comment (jasper-prgh) [inline src/components/container/PartnerBanner/PartnerBanner.js]: what are these two img-tags without src used for?

**Review by jasper-prgh — CHANGES_REQUESTED** [review]

File: src/components/container/PartnerBanner/PartnerBanner.js (hunk 9 above)
Comment (amindu00) [reply src/components/container/PartnerBanner/PartnerBanner.js]: for testing purposes, to see the starting banner and ending banner

Comment (jasper-prgh) [reply src/components/container/PartnerBanner/PartnerBanner.js]: Okay, then remove please. The code we merge into mian should be production ready

**Review by jasper-prgh — CHANGES_REQUESTED** [review]

File: src/components/container/PartnerBanner/PartnerBanner.scss [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+
+    div {
+        display: flex;
+        flex-direction: row;
+        gap: 4vw;
+        animation: slide 10s 1s linear infinite;
+    }
+
+    img {
+        width: 10vw;
```
Comment (jasper-prgh) [inline src/components/container/PartnerBanner/PartnerBanner.scss]: is this responsive? Shouldnt the images have a fixed width?

File: src/components/container/PartnerBanner/PartnerBanner.scss [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+    50% {
+        margin-left: -25%;
+    }
+
+    100% {
+        margin-left: -50%;
+    }
+}
```
Comment (tinkabel85) [inline src/components/container/PartnerBanner/PartnerBanner.scss]: is it a desired behaviour that the  slider will be empty for a while a soon as the last banner is moved to the left? I hope I explained it more or less ok))

Comment (amindu00) [reply src/components/container/PartnerBanner/PartnerBanner.scss]: it half empty because  i wanted to show last logo properly.

Comment (tinkabel85) [reply src/components/container/PartnerBanner/PartnerBanner.scss]: If I reproduce the code correctly,  we have increased/decreased margin size, but no slide effect.
I though we want smth like this https://codesandbox.io/s/distracted-bas-bzekny?file=/src/styles.css
However it is not working smoothly so far... Need to research a bit

Comment (amindu00) [reply src/components/container/PartnerBanner/PartnerBanner.scss]: tbh I was confused about how it worked too. but it loop all the logos. now i think y i worked bcz i saw the bottom scrollbar XD thanks! i try this fix

**Review by amindu00 — COMMENTED** [review]
struggling to loop through all the images on some screens :( nee dhelp

**Review by jasper-prgh — APPROVED** [review]

File: src/components/container/PartnerBanner/PartnerBanner.scss [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+    overflow: hidden;
+    background-color: var(--color-bunting); 
+
+    div {
+        position: relative;
+        display: flex;
+        flex-direction: row;
+        // transform: translateX(-10vw);
+        animation: slide 20s 1s linear infinite;
+
```
Comment (tinkabel85) [inline src/components/container/PartnerBanner/PartnerBanner.scss]: don't we want to add a gap property here to make the prgh logos not so close to each other? and also it prevents from having an empty space at the end of moving, - at least during my tests))

File: src/components/container/App/App.js [hunk 13]
```diff
@@ -1,10 +1,12 @@
 import HeroSection from '../../components/HeroSection/HeroSection';
+import PartnerBanner from '../PartnerBanner/PartnerBanner';
 import './App.scss';
 
 function App() {
   return (
     <div className="App">
       <HeroSection />
+      <PartnerBanner/>
```
Comment (tinkabel85) [inline src/components/container/App/App.js]: should be removed before merge

File: src/components/container/PartnerBanner/PartnerBanner.scss (hunk 12 above)
Comment (Star4ze) [reply src/components/container/PartnerBanner/PartnerBanner.scss:12]: I added margin for the image  :)

File: src/components/container/PartnerBanner/PartnerBanner.js [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+import Logo from "./assets/partnerLogos/prghDarkMain.svg";
+
+function PartnerBanner() {
+  const banners = Array(10).fill({ name: "PRGH", src: Logo });
+
+  const responsive = {
+    desktop: {
+      breakpoint: { max: 3000, min: 1024 },
+      items: 5,
+      slidesToSlide: 2, // optional, default to 1.
```
Comment (tinkabel85) [inline src/components/container/PartnerBanner/PartnerBanner.js:14]: maybe lets remove the comments

File: src/components/container/PartnerBanner/PartnerBanner.scss [hunk 15]
```diff
@@ -0,0 +1,9 @@
+.PartnerBanner {
+    background-color: var(--color-bunting);
+    
+    img {
+        height: 2em;
+        object-fit: contain;
+        margin: 1.25em;
+    }
```
Comment (tinkabel85) [inline src/components/container/PartnerBanner/PartnerBanner.scss:8]: we might change the sizes later on - to match the design, like 
 img {
        height: 1.1em;
        object-fit: contain;
        margin: 1.4em;
    }

**Review by tinkabel85 — APPROVED** [review]

**Review by Puxi19 — APPROVED** [review]

**Pull request merged**